src/globals.c

```diff
@@ -1018,6 +1016,14 @@ void global_release()
     strbuf_free(elf_section);
 }
 
+/* Reports an error without specifying a position */
+void error_no_loc(char *msg)
```

Comment:
Let's rename it to `fatal`. Such helper functions should be always having shorter names.